src/ui/components/index.ts

```diff
@@ -9,3 +9,6 @@ export * from './Select';
 export * from './Input';
 export * from './Table';
 export * from './WhpptLink';
+export * from './Heading';
+export * from './QueryInput';
+export * from './GalleryTag';
```

Comment:
Pretty sure this one can be in the Gallery. Unless its going to be more of a generic tag.